src/pages/LogIn.jsx

```diff
@@ -1,14 +1,76 @@
-import React from 'react';
+import {useRef, useState, useEffect} from 'react';
 
 function LogIn() {
+  const userRef = useRef();
+  const errRef = useRef();
+
+  const [user, setUser] = useState('');
+  const [pwd, setPwd] = useState('');
+  const [errMsg, setErrMsg] = useState('');
+
+useEffect(() => {
+  userRef.current.focus();
+}, [])
+
+useEffect(() => {
+ setErrMsg('');
+}, [user,pwd])
+
+const handleSubmit = async (e) => {
+  e.preventDefault();
+  console.log(user, pwd);
+  setUser('');
+  setPwd('');
+  setSuccess(true);
+}
   return (
+  <>
+    {success ? (
```

Comment:
What is this `success` variable? It isn't defined anywhere in this file

Reply:
My bad, I wanted to push just Register.jsx, not all changes. I am still working on LogIn.jsx, so it's not ready to rewiew